extensions/README.md

```diff
@@ -27,9 +28,11 @@ Current list of extended components:
 | [`amp-carousel`](amp-carousel/amp-carousel.md) | Displays multiple similar pieces of content along a horizontal axis. |
 | [`amp-dailymotion`](amp-dailymotion/amp-dailymotion.md) | Displays a [Dailymotion](https://www.dailymotion.com) video. |
 | [`amp-dynamic-css-classes`](amp-dynamic-css-classes/amp-dynamic-css-classes.md) | Adds several dynamic CSS class names onto the HTML element. |
+| [`amp-embed`](../builtins/amp-embed.md) | Embed elements into the AMP page. Can be used instead of `amp-ad` when semantically more accurate.  |
```

Comment:
This wording could be more concise to clarify that amp-embed is only useful as an alias of amp-ad